project.clj

```diff
@@ -1,15 +1,15 @@
-(defproject frankiesardo/pedestal-swagger "0.4.4-SNAPSHOT"
+(defproject frankiesardo/pedestal-swagger "0.4.4-NUBANK-SNAPSHOT"
```

Comment:
Maybe we should rename it to nu-pedestal-swagger like we did for our faraday fork?

https://github.com/nubank/nu-faraday/blob/master/project.clj#L1